azext_edge/edge/providers/orchestration/upgrade.py

```diff
@@ -204,23 +210,26 @@ def _get_resource_map(self) -> IoTOperationsResourceMap:
                 resource_name=self.instance_name,
                 api_version=INSTANCE_7_API
             )
+            # don't deal with bug bash m2's - only released version
+            if self.instance["properties"]["version"] != "0.7.31":
```

Comment:
This should probably be in a constant - will also make it easier to update later when our version window shifts.

Reply:
True